src/www/ui/scripts/change-license-common.js

```diff
@@ -295,6 +302,12 @@ function closeTextModal() {
   textModal.modal('hide');
 }
 
+function cleanTextModal() {
+  var textAreaValue = document.getElementById("referenceText").value;
+  textAreaValue = textAreaValue.replace(/,/g, '');
+  document.getElementById("referenceText").value = textAreaValue;
+}
```

Comment:
Please try to reuse following function https://github.com/fossology/fossology/blob/36db1bec85d1386d1050e87d9c3aeb1fcc43f103/src/www/ui/scripts/change-license-view.js#L97-L116

A simpler solution will be to pass value for `textField` as a parameter to `cleanText()`. What do you think?

Reply:
Sir, I have made the changes. Kindly check